cloud_service/installing_cloud_ocp/install-secured-cluster-cloud-ocp.adoc

```diff
@@ -14,7 +14,7 @@ You can install {product-title-managed-short} on your secured clusters by using
 * You have created your {osp} cluster and installed the Operator on it.
 * In the ACS Console in {product-title-managed-short}, you have created and downloaded the init bundle.
 * You applied the init bundle by using the `oc create` command.
-* During installation, you noted the *Central API Endpoint*, including the address and the port number. You can view this information by choosing *Advanced Cluster Security* -> *ACS Instances* from the cloud console navigation menu, and then clicking the ACS instance you created.
+* During installation, you noted the *Central API Endpoint* address. You can view this information by choosing *Advanced Cluster Security* -> *ACS Instances* from the cloud console navigation menu, and then clicking the ACS instance you created.
```

Comment:
I see over ten instances of port number with reference to the Central API Endpoint,  in this document but these are OK?

Reply:
Thank you @dfitzmau 🤦🏼‍♂️ Somehow, I missed pushing the latest commit from my machine before asking for a review.

Reply:
No problem, @gaurav-nelson . Thanks for confirming. Let me know if you want me to re-review the conten.